### PR TITLE
modified base structure; added custom component;

### PR DESCRIPTION
**Pull request by alexarnautu**

### CODE REVIEW

File: app/src/app/cs/services/jobs.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
```
Comment (PhilipCotan) [inline app/src/app/cs/services/jobs.service.spec.ts]: Am zis ca facem fara teste, right? Ne mai complicam si cu ele?

File: app/src/app/cs/pages/landing/landing.component.ts [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
-
+import { Component, OnInit } from '@angular/core';
+import { Job } from '../../models/job';
+import { JobsService } from '../../services/jobs.service';
 @Component({
     selector: 'cs-landing-component',
     templateUrl: './landing.component.html',
 })
-export class LandingComponent {
+export class LandingComponent implements OnInit {
```
Comment (PhilipCotan) [inline app/src/app/cs/pages/landing/landing.component.ts]: Asta nu e landing component. Asta e JobsComponent. LandingComponent o sa fie prima pagina la care lucreaza @victortodoran . Cea cu o imagine si cu doua butoane: Search Job, Create Job. Remember?

File: app/src/app/cs/services/jobs.service.spec.ts (hunk 1 above)
Comment (dooma) [reply app/src/app/cs/services/jobs.service.spec.ts]: @PhilipCotan `ng g service/component/etc` generates tests. Please ignore them.

File: app/src/app/cs/pages/landing/landing.component.ts (hunk 2 above)
Comment (dooma) [reply app/src/app/cs/pages/landing/landing.component.ts]: Deci o refactorizam?

Comment (victortodoran) [reply app/src/app/cs/pages/landing/landing.component.ts]: A trecut Arni pe landing. Am facut eu Jobs

Comment (dooma) [reply app/src/app/cs/pages/landing/landing.component.ts]: @PhilipCotan, agree with @victortodoran ?

Comment (PhilipCotan) [reply app/src/app/cs/pages/landing/landing.component.ts]: Agree cu conditia ca NU trebuie sa avem jobs pe componentul asta. Logica de afisare de joburi trebuie stearsa, cu merge-uita pe develop.

File: app/src/app/cs/services/jobs.service.spec.ts (hunk 1 above)
Comment (PhilipCotan) [reply app/src/app/cs/services/jobs.service.spec.ts]: Yes, totally agree with you. But all we have to do is deleting the file, not commit it to develop for no reason but adding confusion and complexity to our folder structure.

**Pull request merged**